src/components/RichTextEditor/Toolbar.tsx

```diff
@@ -3,126 +3,73 @@
-                            <IconTextAlignJustify size={IconSize.Size16} />
-                        </BlockStyleButton>
-
-                        <BlockStyleButton blockType={BlockStyleTypes.Paragraph} textAlign={TextAlignTypes.AlignLeft}>
-                            <IconTextAlignLeft size={IconSize.Size16} />
-                        </BlockStyleButton>
-
-                        <BlockStyleButton blockType={BlockStyleTypes.Paragraph} textAlign={TextAlignTypes.AlignCenter}>
-                            <IconTextAlignCenter size={IconSize.Size16} />
-                        </BlockStyleButton>
-
-                        <BlockStyleButton blockType={BlockStyleTypes.Paragraph} textAlign={TextAlignTypes.AlignRight}>
-                            <IconTextAlignRight size={IconSize.Size16} />
-                        </BlockStyleButton>
-                    </ButtonGroup>
-
-                    <ButtonGroup>
-                        <InlineStyleButton style={InlineStyles.Bold}>
-                            <IconBold size={IconSize.Size16} />
-                        </InlineStyleButton>
-
-                        <InlineStyleButton style={InlineStyles.Italic}>
-                            <IconItalic size={IconSize.Size16} />
-                        </InlineStyleButton>
-
-                        <InlineStyleButton style={InlineStyles.Underline}>
-                            <IconUnderline size={IconSize.Size12} />
-                        </InlineStyleButton>
-
-                        <InlineStyleButton style={InlineStyles.Strikethrough}>s</InlineStyleButton>
-
-                        <InlineStyleButton style={InlineStyles.Code}>
-                            <IconSnippet size={IconSize.Size16} />
-                        </InlineStyleButton>
-
-                        <BlockStyleButton blockType={BlockStyleTypes.OrderedList}>
-                            <IconListNumbers size={IconSize.Size12} />
-                        </BlockStyleButton>
-
-                        <BlockStyleButton blockType={BlockStyleTypes.UnorderedList}>
-                            <IconListBullets size={IconSize.Size16} />
-                        </BlockStyleButton>
-                    </ButtonGroup>
-                </div>
-
-                <div
-                    ref={inlineToolbarArrowRef}
-                    style={styles.arrow}
-                    className="tw-popper-arrow tw-absolute tw-w-2 tw-h-2 tw-pointer-events-none before:tw-absolute before:tw-z-10 before:tw-w-2 before:tw-h-2 before:tw-bg-white before:tw-rotate-45"
-                ></div>
-            </div>
-        </>
+        <div
+            data-test-id="toolbar"
+            className="tw-flex tw-p-0.5 tw-items-center tw-bg-white tw-rounded tw-shadow-mid tw-gap-0.5"
+        >
+            <ButtonGroup testId="text-alignment-buttons">
+                <AlignToolbarButton value="left" icon={<IconTextAlignLeft />} />
```

Comment:
The icons are bigger than in the current RTE, I think you need to specify the correct size

Reply:
Current RTE is not according to Figma anway, so I'll leave it as it is and we can tackle that, as soon as the design is fixed an we implement the actual design of the RTE.